contrib/sync-places.py

```diff
@@ -139,6 +158,7 @@ async def do_dump(session, args):
                 my-place1: # Replace with your place
                   matches: # A list of match patterns. Replace with your match patterns
                     - "*/my-place1/*"
+                    - ["exporter/my-place1/resource", "name"] # named matches supported
```

Comment:
I think it would be more convenient for users if this was done as a dictionary mapping:

```yaml
matches:
  - "*/my-place1/*"
  - name: "exporter/my-place1/resource"
```

Reply:
@JoshuaWatt thanks for the review.
I like the idea of the dictionary, I just need to use the resources as keys as names are not unique

Reply:
It technically doesn't matter because each dictionary is a dedicated list element, so you should only have a single dictionary key-value pair in each dictionary anyway.

I don't really care either way; which ever method makes more sense or has precedence is fine with me

Reply:
@Bastian-Krause what do you think about the dictionary order?
I lean a bit more in favor of {resource: name} as it matches the order of `labgrid-client -v places`

Reply:
OK